src/locales/en.json

```diff
@@ -92,13 +92,13 @@
     "heading": "RECOMMENDED LICENSE",
     "heading-cc0": "RECOMMENDED",
     "full-description": {
-      "cc0": "This license is marked with a CC0 public domain dedication that the user is giving up his copyright and allowing reusers to distribute, remix, adapt, and build upon the material in any medium or format, even for commercial purposes.",
+      "cc0": "By marking the work with a  CC0 public domain dedication, the creator is giving up their copyright and allowing reusers to distribute, remix, adapt, and build upon the material in any medium or format, even for commercial purposes.",
       "cc-by": "This license requires that reusers give credit to the creator. It allows reusers to distribute, remix, adapt, and build upon the material in any medium or format, even for commercial purposes.",
       "cc-by-sa": "This license requires that reusers give credit to the creator. It allows reusers to distribute, remix, adapt, and build upon the material in any medium or format, even for commercial purposes. If others remix, adapt, or build upon the material, they must license the modified material under identical terms. ",
       "cc-by-nd": "This license requires that reusers give credit to the creator. It allows reusers to copy and distribute the material in any medium or format in unadapted form only, even for commercial purposes. ",
       "cc-by-nc": "This license requires that reusers give credit to the creator. It allows reusers to distribute, remix, adapt, and build upon the material in any medium or format, for noncommercial purposes only.",
       "cc-by-nc-sa": "This license requires that reusers give credit to the creator. It allows reusers to distribute, remix, adapt, and build upon the material in any medium or format, for noncommercial purposes only. If others modify or adapt the material, they must license the modified material under identical terms.",
-      "cc-by-nc-nd": "This license requires that reusers give credit to the creator. It allows reusers to copy and distribute the material in any medium or format, for noncommercial purposes only. If others remix, adapt, or build upon the material, they may not distribute the modified material."
+      "cc-by-nc-nd": "This license requires that reusers give credit to the creator. It allows reusers to copy and distribute the material in any medium or format in unadapted form only, even for commercial purposes. "
```

Comment:
This is the text for an NC - Non commercial - license. So, it should state in unadapted form and for noncommercial purposes only.